spec/tictactoe_clojure/game_spec.clj

```diff
@@ -0,0 +1,14 @@
+(ns tictactoe-clojure.game-spec
+  (:require [speclj.core :refer :all]
+            [tictactoe-clojure.human-player :as human-player]
+            [tictactoe-clojure.console :as console]
+            [tictactoe-clojure.game :refer :all]))
+
+(describe "tictactoe-clojure.game"
+  (with new-board [1 2 3 4 5 6 7 8 9])
+  
+  (describe "#play-game"
+    (it "loops through a game where the human enters 1, 2, 4 and loses to the computer"
+      (with-redefs [human-player/human-move (fn [_] (with-in-str "1\n2\n4\n"))
+                    console/display (fn [_] "Player O wins")]
+          (should= "Player O wins" (play-game @new-board "X"))))))
```

Comment:
I briefly spoke to Richard about this and he recommended testing the game engine with two human players. I think this would be a good idea, although this particular PR doesn't support a two player game. Would you have some time to pair tomorrow or Wednesday on this?

Thus far, I refactored the test to check for the following behaviors. It ends the game loop if the board has a winner. It alternates taking turns (I am testing this on a pre-played so I am not relying on the computer move as much). 

 ```
(describe "tictactoe-clojure.game"
  (with x-win-board ["X" "X" "X" 4 "O" 6 7 8 "O"])
  (with x-1-move-from-win-board [1 "X" "X" 4 "O" 6 "X" "X" 9])
  
  (describe "#play-game"
    (it "end the game loop and displays the winner if someone wins"
      (with-redefs [rules/game-over? (fn [_] true)
                    console/display (fn [_] "Player O wins")]
          (should= "Player O wins" (play-game @x-win-board "X"))))
    (it "loops through a game once and takes two turns"
      (with-redefs [human-player/human-move (fn [_] 1)
                    console/display (fn [_] "Player X wins")]
        (should= "Player X wins" (play-game @x-1-move-from-win-board "O"))))))
```